packrat.js

```diff
@@ -230,14 +230,16 @@ Packrat.prototype.runCommand = function() {
         this.log('Packrat is running `%s`...', command);
     }
 
-    result = sh.exec(command, true);
+    result = spawnSync('sh', [ '-c', command ], {
+        stdio: [ 0, 1, 2 ]
+    });
 
     if (result.stdout) {
-        this.log(result.stdout);
+        this.log(String(result.stdout));
     }
 
-    if (result.code !== 0) {
-        this.onError(new Error(util.format('`%s` command failed', command)), result.code);
+    if (result.status !== 0) {
+        this.onError(new Error(util.format('`%s` command failed', command)), result.status);
     }
 
     return result;
```

Comment:
But (which is worse) `.stdout` of returned `result` is used at [here](https://github.com/isqua/packrat/blob/spawnSync/packrat.js#L180), so the install log is `null`, of course.

Actually, this "install log" thing is the only place, where we need to get stdout of `runCommand`, so I guess we can use smth like `%install command% | tee %installLogLocation%`. Mind making this change?